spec/flows/steps/helper_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe Steps::Helper do
+  describe "#relevant_step?" do
+    context "when there is asylum support data plus income data" do
+      let(:session_data) do
+        {
+          "immigration_or_asylum_type_upper_tribunal" => "immigration_upper",
+          "asylum_support" => true,
+          "employment_status" => "in_work",
+          "incomes" => [
+            {
+              "income_type" => "employment",
+              "income_frequency" => "monthly",
+              "gross_income" => 100,
+              "income_tax" => 20,
+              "national_insurance" => 3,
+            },
+          ],
+        }
+      end
+
+      it "returns the correct relevant step" do
```

Comment:
I assume this is required for coverage? hence the corresponding addition in en.yml.

Reply:
I didn't add these specs for the coverage so much, but just to check the new method was returning directly what I expected it to when the data was looking incohesive (thinking about changing answers). I wanted to test the method directly rather than with all the moving parts of a flow test

Reply:
@HettieS I found the issue, essentially in ClientAssetsForm which calls BankAccountModel :smod_applicable is being returned as true. The simple fix is to add 
        immigration_or_asylum_type_upper_tribunal: "immigration_upper",
          asylum_support: false,
          
       to the test setup, just before you set empoloyment status. In the test setup you had assets was a valid step and smod was applicable. we just need smod to not be applicable because really we dont care at the point that EE is called.